src/scenes/Event/components/ResultHistory/index.js

```diff
@@ -74,33 +81,38 @@ export default class EventResultHistory extends Component {
               </TableRow>
             </TableHead>
             <TableBody>
-              {_.map(sortedOracles, (oracle, index) => {
-                let invalidOption = 'Invalid';
-                if (oracle.localizedInvalid !== undefined) {
-                  invalidOption = oracle.localizedInvalid.parse(intl.locale);
-                }
-                return (
-                  <TableRow key={`result-${index}`} selected={index % 2 === 1}>
-                    <TableCell padding="dense">{getShortLocalDateTimeString(oracle.endTime)}</TableCell>
-                    <TableCell padding="dense">{this.getTypeText(oracle, index)}</TableCell>
-                    <TableCell padding="dense">
-                      {(currentEvent.phase === Phases.VOTING || index !== filteredOracles.length - 1) && index !== 0
-                        ? `#${oracle.resultIdx + 1} ${oracle.options[oracle.resultIdx].name === 'Invalid' ? invalidOption : oracle.options[oracle.resultIdx].name}`
-                        : ''
-                      }
-                    </TableCell>
-                    <TableCell padding="dense">{`${_.sum(oracle.amounts)} ${oracle.token}`}</TableCell>
-                  </TableRow>
-                );
-              })}
+              <ResultRows sortedOracles={sortedOracles} getTypeText={this.getTypeText} {...this.props} />
             </TableBody>
           </Table>
-        ) : (
-          <Typography variant="body1">
-            <FormattedMessage id="str.emptyTxHistory" defaultMessage="You do not have any transactions right now." />
```

Comment:
Do we need to have this no transactions part ?

Reply:
Decided to just remove it entirely.
1) it was the wrong text.
2) thought it wasn't relevant to show this message for the user. thought it would just confuse them more.
3. It is a bug if they see this message since we intentionally start showing the result history from the first arbitration round and on.